<launch>

    <node pkg='path_tracking_py' name='path_tracking' type='tracking_r.py' output="screen">
        <!-- <remap from="/cmd_vel" to="/move_base/cmd_vel"/> -->
    </node>

</launch>